x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java

```diff
@@ -2650,10 +2650,56 @@ public void testReportingUserRole() {
         assertThat(roleDescriptor.getMetadata(), hasEntry("_reserved", true));
         assertThat(roleDescriptor.getMetadata(), hasEntry("_deprecated", true));
 
+        final String applicationName = "kibana-.kibana";
+
+        final Set<String> applicationPrivilegeNames = Set.of(
+            "feature_discover.minimal_read",
+            "feature_discover.generate_report",
+            "feature_dashboard.minimal_read",
+            "feature_dashboard.generate_report",
+            "feature_dashboard.download_csv_report",
+            "feature_canvas.minimal_read",
+            "feature_canvas.generate_report",
+            "feature_visualize.minimal_read",
+            "feature_visualize.generate_report"
+        );
+
+        final Set<String> allowedApplicationActionPatterns = Set.of(
+            "login:",
```

Comment:
These are only test examples of allowed actions and are not ment to represent the accurate list of actions.